### PR TITLE
Styling the Landing Page

### PR DESCRIPTION
**Pull request by sruthin21**
## What Does This PR Do??
This PR add's styling to the landing page

Screenshot (386)
Screenshot (387)
Screenshot (388)
Screenshot (389)

[get-amigo-github-io-seven.vercel.app](https://get-amigo-github-io-seven.vercel.app/)
Deploy link to my PR

### CODE REVIEW

File: package-lock.json
Comment (Bhaumik-Tandan) [inline package-lock.json:1]: please remove this file we are using pnpm

File: src/components/landing/nav.tsx [hunk 1]
```diff
@@ -6,23 +6,42 @@ import { Bars3Icon, XMarkIcon } from "@heroicons/react/24/outline";
...
+  },
+  {
+    name: "About",
+    href: "#",
+  },
+  {
+    name: "Contact",
+    href: "#",
+  },
+];
```
Comment (Bhaumik-Tandan) [inline src/components/landing/nav.tsx:26]: what is use of nav bar if it does not do anything

**Review by Bhaumik-Tandan — CHANGES_REQUESTED** [review]

File: yarn.lock
Comment (Bhaumik-Tandan) [inline yarn.lock:1]: There should be only one lock file in a project

**Review by Bhaumik-Tandan — CHANGES_REQUESTED** [review]

File: src/components/landing/nav.tsx (hunk 1 above)
Comment (sruthin21) [reply src/components/landing/nav.tsx]: > what is use of nav bar if it does not do anything

Ok Remove navigation And Navbar

File: yarn.lock
Comment (sruthin21) [reply yarn.lock:1]: Lock File Means??

Comment (sruthin21) [reply yarn.lock:1]: Which lock file should be deleted

**Comment by Bhaumik-Tandan** [comment]
please make a pr towards dev

File: yarn.lock
Comment (Bhaumik-Tandan) [reply yarn.lock:1]: both since we are using pnpn

Comment (sruthin21) [reply yarn.lock:1]: yarn.lock and package-lock.json Is this the two lock files

**Comment by sruthin21** [comment]
> please make a pr towards dev

how to make pr towards dev

**Comment by mrprince88** [comment]
We dont need any navigation and login buttons here @sruthin21. Can you remove these and also test for responsiveness.

Thanks

**Pull request closed without merging**